qemu/tests/qemu_img_lock_reject_boot.py

```diff
@@ -32,6 +34,9 @@ def _verify_write_lock_err_msg(test, img_file=None):
         test.log.info("Verify qemu-img write lock err msg.",)
         msgs = ['"write" lock',
                 'Is another process using the image']
+        # Avoid timing issues between writing to log and the check itself
+        check_lock_timeout = int(params["check_lock_timeout"])
```

Comment:
May be more simple that  `time.sleep(5)`  ^^

If we keep the parameter,  it can be:
```
check_lock_timeout = params.get_numeric('check_lock_timeout', 5)
time.sleep(check_lock_timeout)
```

Both are ok for me ~

Reply:
Yes, I agree. This is better for sure, thanks :+1: (Fixed.)

Reply:
JOB LOG    : /root/avocado/job-results/job-2023-06-07T04.16-e1a8af4/job.log
 (1/1) Host_RHEL.m9.u2.v0.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.qemu_img_lock.reject_boot_same_img_twice: STARTED
 (1/1) Host_RHEL.m9.u2.v0.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.qemu_img_lock.reject_boot_same_img_twice: PASS (55.84 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0